<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>912</width>
    <height>561</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Single trivia game</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="question_lbl">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>30</y>
     <width>511</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>question</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>140</y>
     <width>511</width>
     <height>221</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <column>
    <property name="text">
     <string>answer</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="select_btn">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>380</y>
     <width>80</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Select</string>
   </property>
  </widget>
  <widget class="QPushButton" name="next_btn">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>380</y>
     <width>80</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_reply">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>370</y>
     <width>331</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>reply</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_correct_reply">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>440</y>
     <width>351</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>reply</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_correct">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>440</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Correct answer:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="leave_btn">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>380</y>
     <width>80</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Leave</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
